pom.xml

```diff
@@ -84,4 +104,4 @@
             </plugins>
         </pluginManagement>
     </build>
-</project>
+</project>
```

Comment:
Add empty line at the end of the file)

Reply:
> 
I don't know how it happened, my bad.
<img width="399" alt="image" src="https://user-images.githubusercontent.com/98731880/178224091-ad2770c7-ba07-427e-bdd2-fbf351e90388.png">